tests/test_problems.py

```diff
@@ -0,0 +1,53 @@
+import sqlite3
+
+import pytest
+
+from sirenity.euler import ProblemManager
+from sirenity.euler.problem import Problem
+
+# Clear db
+with open("test.db", "w") as f:
+    f.write("")
+
+manager = ProblemManager("test.db")
+
+
+class TestClass:
```

Comment:
```suggestion
class TestProblemManager:
```

The name of the test class should describe what it's testing, perhaps `TestProblemManager` as a name?